CMakeLists.txt

```diff
@@ -183,3 +190,6 @@ add_subdirectory(mgmt/utils)
 add_subdirectory(mgmt/config)
 add_subdirectory(mgmt/rpc)
 add_subdirectory(src/traffic_server)
+add_subdirectory(plugins)
+
+add_subdirectory(configs)
```

Comment:
https://stackoverflow.com/a/44704969/629530

Reply:
I had this for vim, but been using clion lately.  I'll look up the equivalent.